File: android/app/src/main/java/tut/flightbookingsystem/model/MealDrink.java
package tut.flightbookingsystem.model;

import java.io.Serializable;

public class MealDrink implements Serializable {
    public long id;
    public long passenger_id;
    public long drink_id;
    public Drink drink;
    public String created_at;
    public String updated_at;
}

File: web/app/MealDrink.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class MealDrink extends Model
{

  protected $fillable=[
    'passenger_id', 'drink_id', 'meal_id'
  ];

  public function passenger(){
    return $this->belongsTo('App\Passenger', 'id', 'passenger_id');
  }
  public function drink(){
    return $this->hasOne('App\Drink', 'id', 'drink_id');
  }
}
